joi-validations/topics.validations.js

```diff
@@ -5,6 +5,16 @@ const TopicValidation = {
     query: {
       name: string.required()
     }
+  },
+  checkName: {
+    query: {
+      name: string.min(3).required()
+    }
+  },
+  create: {
+    body: {
+      name: string.min(3).required()
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the validation rules for `checkName` and `create` are good as they enforce a minimum length requirement of 3 characters for the `name` parameter. However, it might be beneficial to also include a maximum length constraint to prevent excessively long names which could lead to potential issues with storage or display. Here's how you can add a max length constraint:

```diff
  checkName: {
    query: {
      name: string.min(3).max(50).required()
    }
  },
  create: {
    body: {
      name: string.min(3).max(50).required()
    }
```
This will limit the `name` length to 50 characters. Adjust the number as per your requirements.


<!-- This is an auto-generated comment by OSS CodeRabbit -->